apps/antalmanac/tests/schedule.test.tsx

```diff
@@ -0,0 +1,16 @@
+import { describe, test, expect } from 'vitest';
+import { render } from '@testing-library/react';
+import { Schedules } from '../src/stores/Schedules';
```

Comment:
Just because we have aliases, they could be used for shorter import paths.

```suggestion
import { Schedules } from '$stores/Schedules';
```